packages/vx-tooltip/src/tooltips/Tooltip.tsx

```diff
@@ -1,16 +1,22 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import cx from 'classnames';
 
-Tooltip.propTypes = {
-  left: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
-  top: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
-  className: PropTypes.string,
-  style: PropTypes.object,
-  children: PropTypes.any,
+export type TooltipProps = {
+  left?: number;
+  top?: number;
+  className?: string;
+  style?: React.CSSProperties;
+  children?: React.ReactNode;
 };
 
-export default function Tooltip({ className, top, left, style, children, ...restProps }) {
+export default function Tooltip({
+  className,
+  top,
+  left,
+  style,
+  children,
+  ...restProps
+}: TooltipProps) {
```

Comment:
missed this one earlier, but we'd like to preserve the ability to pass additional `restProps` here so we don't make breaking changes. the pattern I think we've aligned on is updating this line to `TooltipProps & JSX.IntrinsicElements.div` so that `TooltipProps` can still be imported separately but the component accepts any `div` props as well.

Reply:
oh, you are right! It's a great point, thanks!